<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Open|Close main navigation</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KeySequenceLineEdit" name="openCloseMainNavigationLineEdit" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Open|Close call monitor</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KeySequenceLineEdit" name="openCloseCallMonitorLineEdit" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Open|Close detail window</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KeySequenceLineEdit" name="openCloseDetailWindowLineEdit" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Open|Close all windows</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KeySequenceLineEdit" name="openCloseAllWindowsLineEdit" native="true"/>
       </item>
       <item row="4" column="1">
        <widget class="KeySequenceLineEdit" name="openHelpLineEdit" native="true"/>
       </item>
       <item row="5" column="1">
        <widget class="KeySequenceLineEdit" name="openPhoneLineEdit" native="true"/>
       </item>
       <item row="6" column="1">
        <widget class="KeySequenceLineEdit" name="openContactsLineEdit" native="true"/>
       </item>
       <item row="7" column="1">
        <widget class="KeySequenceLineEdit" name="openCallHisotryLineEdit" native="true"/>
       </item>
       <item row="8" column="1">
        <widget class="KeySequenceLineEdit" name="holdUnholdCallLineEdit" native="true"/>
       </item>
       <item row="9" column="1">
        <widget class="KeySequenceLineEdit" name="startStopCallRecordingLineEdit" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Open Help</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Open phone settings</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Open contacts</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Open call history</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Hold / Unhold a call (active line)</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Start / stop call recording</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Take a call</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>HangUp a call</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Redial last phone number</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="KeySequenceLineEdit" name="takeCallLineEdit" native="true"/>
       </item>
       <item row="11" column="1">
        <widget class="KeySequenceLineEdit" name="hangUpCallLineEdit" native="true"/>
       </item>
       <item row="12" column="1">
        <widget class="KeySequenceLineEdit" name="redialLastPhoneNumberLineEdit" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mute on / off</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KeySequenceLineEdit" name="muteOnOffLineEdit" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Dial selected area</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Turn up microphone</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Turn down microphone</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Turn up speaker</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Turn down speaker</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KeySequenceLineEdit" name="dialSelectedAreaLineEdit" native="true"/>
       </item>
       <item row="2" column="1">
        <widget class="KeySequenceLineEdit" name="turnUpMicrophoneLineEdit" native="true"/>
       </item>
       <item row="3" column="1">
        <widget class="KeySequenceLineEdit" name="turnDownMicrophoneLineEdit" native="true"/>
       </item>
       <item row="4" column="1">
        <widget class="KeySequenceLineEdit" name="turnUpSpeakerLineEdit" native="true"/>
       </item>
       <item row="5" column="1">
        <widget class="KeySequenceLineEdit" name="turnDownSpeakerLineEdit" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KeySequenceLineEdit</class>
   <extends>QWidget</extends>
   <header location="global">KeySequenceLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
